packit/constants.py

```diff
@@ -128,18 +128,18 @@
     "-D",
     "__scm_apply_git_am(qp:m:) "
     "%{__git} am %{-q} %{-p:-p%{-p*}} && "
-    "patch_name=`basename %{1}` "
-    "commit_msg=`%{__git} log --format=%B -n1` "
+    "patch_name=`basename %{1}` && "
+    "commit_msg=`%{__git} log --format=%B -n1` && "
```

Comment:
If I understand correctly, you want the whole command (setting up macros) to fail if you get an error for either of those variables?

Reply:
correct! because it means we'd get inconsistent results - a malformed source-git repo, failing early should be the way here I'd say